packages/web-react/src/components/Button/Button.tsx

```diff
@@ -33,7 +35,10 @@ const _Button = <T extends ElementType = 'button', C = void, S = void>(
       ref={ref}
       className={classNames(classProps, styleProps.className)}
       style={styleProps.style}
-    />
+    >
```

Comment:
When the button is loading, should it be also disabled? 🤔 Same for the `ButtonLink`.

Reply:
Or for this type of behavior is responsible the developer?

Reply:
Ok, I saw that this is done.